dogowner/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+
+class Dogowner(models.Model):
+
+    Gender = (
+        ('M', 'Male'),
+        ('F', 'Female'),
+        )
+
+    email = models.EmailField(max_length=254)
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    user_name = models.CharField(max_length=50)
+    dog_name = models.CharField(max_length=50)
+    dog_race = models.CharField(max_length=50)
+    dog_age = models.IntegerField()
+    dog_weight = models.FloatField()
+    dog_gender = models.CharField(
+        max_length=1,
+        choices=Gender,
+        default='M',)
+
+# should implement dog_picture
```

Comment:
I suggest using an image url. don't try to store photos in the db, It'll only create a mess.